videostream.js

```diff
@@ -51,20 +52,14 @@ function VideoStream (file, mediaElem, opts) {
 	})
 }
 
-inherits(VideoStream, EventEmitter)
-
 VideoStream.prototype._pump = function () {
 	var self = this
 	self._tracks.forEach(function (track) {
-		pump(track.muxed, track.mediaSource, function (err) {
-			if (err && err.message !== 'premature close') {
```

Comment:
This line used to cause `'premature close'` errors to be silently ignored. With this PR, these errors will be handled and cause the media element to enter an error state.

Is this okay?